lib/routes/route.js

```diff
@@ -1233,45 +1157,41 @@ module.exports = function(app, router) {
                 }
 
                 if (!result) {
-                    routeError(next, {
+                    return sendError(next, {
                         errorID: "oldPasswordIncorrect",
                         message: "Could not change password. Old password is not correct.",
                         field: "oldPassword",
                         statusCode: 400
                     });
-                    return;
                 }
 
                 if (req.body.newPassword !== req.body.newPasswordConfirm) {
-                    routeError(next, {
+                    return sendError(next, {
                         errorID: "passwordsDoNotMatch",
                         message: "Could not change password. Passwords don't match.",
                         fields: ["newPassword", "newPasswordConfirm"],
                         statusCode: 400
                     });
-                    return;
                 }
 
                 if (req.body.newPassword === req.body.oldPassword) {
-                    routeError(next, {
+                    return sendError(next, {
                         errorID: "passwordSame",
                         message: "Could not change password. New password is the same as the old password.",
                         fields: ["oldPassword", "newPassword", "newPasswordConfirm"],
                         statusCode: 400
                     });
-                    return;
                 }
 
                 let passwordStrengthResult = passwordUtil.verifyPasswordStrength(req.body.newPassword);
 
                 if (!passwordStrengthResult.strong) {
-                    routeError(next, {
+                    return sendError(next, {
                         errorID: "passwordNotStrong",
```

Comment:
Might be a good idea to have an enum type for error IDs. That way, error IDs can be made more consistent. (e.g. `passwordNotStrong` here vs. `weakPassword` in /register route.

Reply:
Will take care of this in the future.